### PR TITLE
Improve app strings

### LINKED ISSUE #436: Improve strings used in app
One of the findings in the design thesis was that the texts used in the app are hard to understand and weirdly written sometimes.

### PR DESCRIPTION
**Pull request by SebastianWagner2**
Closes #436 

Also deletes all unused strings defined in app_en.arb

- [X] I have read the [code-style guidelines](https://github.com/hpi-dhc/PharMe/blob/main/docs/FLUTTER_STYLE.md) and confirmed that this PR upholds said guidelines

---

### CODE REVIEW

File: app/lib/l10n/app_en.arb [hunk 1]
```diff
@@ -1,32 +1,20 @@
...
-  "faq_page_description": "On this page we will answer common questions regarding PGx",
+  "faq_page_description": "Here you can find answers to common questions about PGx",
   "faq_pharmacogenomics": "Pharmacogenomics",
 
   "general_appName": "PharMe",
   "general_continue": "Continue",
   "general_retry": "Retry",
   
-  "medications_page_disclaimer": "This assessment is only based on your genomic data. It does not consider other factors such as weight, age and pre-existing conditions.",
+  "medications_page_disclaimer": "This assessment is ONLY based on your genetics. Important factors like weight, age and pre-existing conditions are not considered.",
```
Comment (Toalaah) [inline app/lib/l10n/app_en.arb:17]: This all caps seems a little childish IMO. Also would change second sentence slightly + add oxford comma:

```suggestion
  "medications_page_disclaimer": "This assessment is only based on your genetics. Important factors such as weight, age, or pre-existing conditions are not considered.",
```

File: app/lib/l10n/app_en.arb [hunk 2]
```diff
@@ -41,14 +29,14 @@
   "medications_page_header_guideline": "Clinical Guideline",
   "medications_page_header_recommendation": "Recommendation",
   "medications_page_info_big_i": "i",
-  "medications_page_no_guidelines_for_phenotype": "No guidelines found matching your genetic profile.",
+  "medications_page_no_guidelines_for_phenotype": "We couldn't find a guideline matching your genetics.",
```
Comment (Toalaah) [inline app/lib/l10n/app_en.arb:32]: More formal.

```suggestion
  "medications_page_no_guidelines_for_phenotype": "We could not find a guideline matching your genetics.",
```

File: app/lib/l10n/app_en.arb [hunk 3]
```diff
@@ -41,14 +29,14 @@
...
   "medications_page_sources_cpic_name": "CPIC",
   "medications_page_sources_cpic_description": "Guidelines published by the Clinical Pharmacogenetics Implementation Consortium (CPIC)",
   "medications_page_sources_pharmGkb_name": "PharmGKB",
   "medications_page_sources_pharmGkb_description": "Curated pharmacogenetic studies for this drug-gene reaction",
-  "medications_page_tooltip_classification": "The better the evidence level, the more scientifically accepted is the clinical annotation it refers to. Many studies and good studies mean a high evidence level.",
-  "medications_page_tooltip_further_info": "Get to know the source material! CPIC, DPWG and PharmGKB are well-known for their clinical guidelines regarding drug-gene-interactions. Also see the PGx-Tab for more information ",
-  "medications_page_tooltip_guideline": "Clinical guidelines are designed to help you understand how genetic test results should be used to optimize drug therapy.",
+  "medications_page_tooltip_classification": "The evidence level of a guideline represents how accepted it is by science. Guidelines with a lot of good studies supporting them have a high evidence level.",
```
Comment (Toalaah) [inline app/lib/l10n/app_en.arb:37]:
```suggestion
  "medications_page_tooltip_classification": "The evidence level of a guideline represents how accepted it is by science. Guidelines supported by many reputable publications have a higher evidence level.",
```

File: app/lib/l10n/app_en.arb [hunk 4]
```diff
@@ -63,14 +51,14 @@
   "onboarding_2_text": "Your body reacts to medications uniquely.\n\nMedications that are effective for a majority of people can be toxic to you.",
   "onboarding_3_header": "Genome power unlocked to improve human health",
   "onboarding_3_text": "PharMe warns about unintended drug responses according to your genome.\nThis enables you to avoid medications that are ineffective or have side-effects.",
-  "onboarding_3_disclaimer": "It does not recommend medications for specific symptoms or illnesses!",
+  "onboarding_3_disclaimer": "We DO NOT recommend medications for specific symptoms or illnesses!",
```
Comment (Toalaah) [inline app/lib/l10n/app_en.arb:54]: Again the all-caps thing but I guess at least you are being consistent. At this point no strong opinion I guess, so will leave this up to you.

File: app/lib/l10n/app_en.arb [hunk 5]
```diff
@@ -63,14 +51,14 @@
...
   "onboarding_4_header": "Tailored to your genome",
-  "onboarding_4_text": "For PharMe to work we need your genetic data. But we don’t provide such tests and you cannot make an account. This is why you need to get a test done at a labratory.",
+  "onboarding_4_text": "For PharMe to work, you need to get your genetics tested at a lab. You don't need an account to use PharMe: You can just sign in to the lab's website through our app.",
```
Comment (Toalaah) [inline app/lib/l10n/app_en.arb:56]:
```suggestion
  "onboarding_4_text": "For PharMe to work, you must get your genetics tested at a lab. You do not need an account to use PharMe: Just sign in directly to your lab through our app in order to transfer your data.",
```

File: app/lib/l10n/app_en.arb [hunk 6]
```diff
@@ -63,14 +51,14 @@
...
   "onboarding_4_button": "Find out more about gene tests here.",
   "onboarding_5_header": "We care about your data protection",
-  "onboarding_5_text": "When using PharMe, your genetic data is sent from the lab that took the test straight onto your phone.\n\nOur servers never get to see this data and even our app doesn’t know your identity, only your genetics.",
+  "onboarding_5_text": "After signing in to the lab, your genetic data is sent straight onto your phone.\n\nOur servers know nothing about you, neither your identity nor your genetics.",
```
Comment (Toalaah) [inline app/lib/l10n/app_en.arb:59]:
```suggestion
  "onboarding_5_text": "After signing in to your chosen lab, your genetic data is transferred directly to your phone.\n\nOur servers never see any personally identifiable information. This includes both your identity and your genetic information.",
```

File: app/lib/l10n/app_en.arb [hunk 7]
```diff
@@ -63,14 +51,14 @@
...
     
-  "reports_page_disclaimer_text": "All medications shown here should be used with extreme caution, used in different doses or not be used at all. Check specific medications for details.",
+  "reports_page_disclaimer_text": "All drugs listed here should be used with high caution. Consult your physician with these results before taking one of them. Click on a specific medication for details.",
```
Comment (Toalaah) [inline app/lib/l10n/app_en.arb:61]:
```suggestion
  "reports_page_disclaimer_text": "All drugs listed here should be used with high caution. Consult your physician with these results before taking one of them. Click on a specific medication to view more details.",
```

**Review by Toalaah — CHANGES_REQUESTED** [review]

**Review by LeonHermann322 — APPROVED** [review]

**Review by Toalaah — APPROVED** [review]

**Pull request merged**